### PR TITLE
[12.0] [MIG] send_message_popup

### PR DESCRIPTION
**Pull request by i-vyshnevska**
migration to v12 with improvements
improvements backported 
- https://github.com/OCA/web/pull/1342

### CODE REVIEW

**Review by tschanzt — APPROVED** [review]

File: web_send_message_popup/static/src/js/chatter.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* Copyright 2019 Camptocamp SA
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+odoo.define('web_send_message_popup.Chatter', function (require) {
+    "use strict";
+
+    var Chatter = require('mail.Chatter');
+
+    Chatter.include({
+        _onOpenComposerMessage: function () {
```
Comment (Tardo) [inline web_send_message_popup/static/src/js/chatter.js:9]: I think that you can use:
```js
_onOpenComposerMessage: function () {
    this._super.apply(this, arguments);
    this.suggested_partners_def.done($.proxy(function () {
        this._closeComposer(true);
        this._composer._onOpenFullComposer();
    }, this));
},
```

**Review by Tardo — CHANGES_REQUESTED** [review]

File: web_send_message_popup/static/src/js/chatter.js (hunk 1 above)
Comment (i-vyshnevska) [reply web_send_message_popup/static/src/js/chatter.js:9]: @Tardo thanks for improvement, works perfectly

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@Tardo fine for you? :)

**Review by grindtildeath — APPROVED** [review]
LGTM :+1:

**Review by Tardo — APPROVED** [review]
LGTM :+1:, code review + tested on runbot

**Pull request closed without merging**